### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by taldr27**
### In this milestone I:
- Added a method that takes the first user in the DB.
- Added forms to add new posts, add new comments, and likes.
- Added a method for simulating the first user creating new posts

gif

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   get '/users/:user_id/posts', to: 'posts#index', as: 'post_path'
   get '/users/:user_id/posts/:id', to: 'posts#show', as: 'post_show'
-
+  post '/users/:user_id/posts/:id/comments', to: 'comments#create', as: 'comment_create'
```
Comment (1GORDON) [inline config/routes.rb:4]: Great job @taldr27 🤝

- [x] There should be a pair of routes in the config/routes.rb file for creating comments.
comment-route

- [x] There should be a form in the views/comments/new.html.erb file which accepts input for creating comments.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @taldr27 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (taldr27) [reply config/routes.rb:4]: @1GORDON Hello, thanks for your review, I have a question to solve this problem, do I have to create a new page, like with a new post?

Comment (1GORDON) [reply config/routes.rb:4]: Hi @taldr27 ,
- Just like you created a form for posts. You can also create a form form comments.

**Review by descholar-ceo — APPROVED** [review]
Hi @taldr27 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented the requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**